.cmake-format.py

```diff
@@ -0,0 +1,257 @@
+# ----------------------------------
+# Options affecting listfile parsing
+# ----------------------------------
+with section("parse"):
+
+  # Specify structure for custom cmake functions
+  additional_commands = {
+    'install': {
+      'flags': [],
+      'kwargs': {
+        'TARGETS': '*',
+        'DIRECTORY': 3,
+        'REGEX': 2,
+        'BUNDLE': 2,
+        'RUNTIME': 2,
+        'ARCHIVE': 2,
+        'LIBRARY': 2,
+        'INCLUDES': 2,
+      },
+    },
+  }
+
+  # Override configurations per-command where available
+  override_spec = {}
+
+  # Specify variable tags.
+  vartags = []
+
+  # Specify property tags.
+  proptags = []
+
+# -----------------------------
+# Options affecting formatting.
+# -----------------------------
+with section("format"):
+
+  # Disable formatting entirely, making cmake-format a no-op
+  disable = False
+
+  # How wide to allow formatted cmake files
+  line_width = 80
```

Comment:
In my opinion, in general a line width of 100 looks better in many cases (especially because wrapping heuristics need to be used less often), but I don't have a strong opinion.

Reply:
Generally I agree, but since we already use a width of 80 in all the hundrets of other files, I like to use the same width also for the project files.

Reply:
I don't think our source code files need to match the build system files, but it's your choice.